daemon.py

```diff
@@ -242,16 +242,17 @@ def sync_get_rules(ctx):
         vid = event.object.name
 
         storage_class_name = event.object.annotations.get('volume.beta.kubernetes.io/storage-class')
+        logger.debug('storage class name: {}', storage_class_name)
 
-        if not storage_class_name:
-            logger.debug('Skipping volume with no StorageClass.')
+        if not storage_class_name or storage_class_name == 'default':
```

Comment:
This is the main fix.  The redis volumes were added with default storage class.  We can probably change things later to use the default storage class, but right now it's a good filter for the ones that were done differently and don't need backups.